model/oracledb/registry.yaml

```diff
@@ -0,0 +1,42 @@
+groups:
+  - id: registry.oracledb
+    type: attribute_group
+    display_name: Oracle Database Attributes
+    brief: >
+      This section defines attributes for Oracle Database.
+    attributes:
+      - id: oracledb.implicit_release
```

Comment:
we avoid adding attributes without referencing them on the spans, metrics, events, resources, etc. 
Is there a span/metrics that would use this attribute? Could you define it?

Otherwise, let's postpone adding this and other attributes until we're ready to define signals they should be set on.

Reply:
Yes this attribute  `oracle.db.client.connection.implicit_release` will appear in span, when a query is executed on the connection and an implicit release of connection is done. It has only value `true` and if there is no implicit release of connection, it won't appear in the span. 

[node-oracledb](https://github.com/oracle/node-oracledb) @6.7.0 onwards has code to add this information passed to OT [module](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2612) to add it in span. 

I added the attribute in `model/database/spans.yaml `.Please suggest if this looks fine?